crawl-ref/source/dat/strings/de/messages.txt

```diff
@@ -9401,10 +9401,10 @@ Ihr spürt die alte Wut Eurer Axt.
 %s fällt aus der Luft.
 %%%%
 The tentacle is hauled back through the portal!
-Der Tentakel wird durch das Portal zurückgeschleppt!
+Das Tentakel wird durch das Portal zurückgeschleppt!
 %%%%
 With a roar, the tentacle is hauled back through the portal!
-Mit einem Gebrüll wird die Tentakel zurück durch das Portal gezogen!
+Mit einem Gebrüll wird das Tentakel zurück durch das Portal gezogen!
```

Comment:
This should be:
Mit lautem Getöse wird das Tentakel in das Portal zurückgerissen!

Reply:
done